src/platform/Zephyr/BLEManagerImpl.cpp

```diff
@@ -235,33 +238,45 @@ struct BLEManagerImpl::ServiceData
 
 CHIP_ERROR BLEManagerImpl::StartAdvertising(void)
 {
-    int err = 0;
+    // Define advertising data
+    static ServiceData serviceData;
+    static bt_data advertisingData[2];
+    static bt_data scanResponseData[1];
+    static_assert(sizeof(serviceData) == 10, "Unexpected size of BLE advertising data!");
+
+    chip::Encoding::LittleEndian::Put16(serviceData.uuid, UUID16_CHIPoBLEService.val);
+    ReturnErrorOnFailure(ConfigurationMgr().GetBLEDeviceIdentificationInfo(serviceData.deviceIdInfo));
 
-    // At first run always select fast advertising, on the next attempt slow down interval.
-    const uint32_t intervalMin = mFlags.Has(Flags::kFastAdvertisingEnabled) ? CHIP_DEVICE_CONFIG_BLE_FAST_ADVERTISING_INTERVAL_MIN
-                                                                            : CHIP_DEVICE_CONFIG_BLE_SLOW_ADVERTISING_INTERVAL_MIN;
-    const uint32_t intervalMax = mFlags.Has(Flags::kFastAdvertisingEnabled) ? CHIP_DEVICE_CONFIG_BLE_FAST_ADVERTISING_INTERVAL_MAX
-                                                                            : CHIP_DEVICE_CONFIG_BLE_SLOW_ADVERTISING_INTERVAL_MAX;
+    mAdvertisingRequest.priority    = 0;
```

Comment:
We set commissioning advertising to the highest priority (0) and  the BLE SMP advertising to the lowest possible (255) because we only have 2 BLE services (so that we can extend this to have some middle priorities later on), right? But we may as well set these to 0 and 1 and it will work the same? This is just to backup my understanding.

Reply:
Yes, I think I may introduce a configuration constant for CHIP BLE advertising to allow a user to change it.